docs/api/version3.md

```diff
@@ -2117,6 +2117,49 @@ https://service.lisk.com/api/v3/nft
 ```
 
 
+
+### NFT supported
```

Comment:
Could you please add the relevant entry to the ToC at the top of the file?

Reply:
Applied.